Terminal.Gui/Core/View.cs

```diff
@@ -446,70 +446,102 @@ void SetTabIndex ()
 		public virtual bool WantContinuousButtonPressed { get; set; }
 
 		/// <summary>
-		/// Gets or sets the frame for the view. The frame is relative to the view's container (<see cref="SuperView"/>).
+		/// Gets or sets the location and size of the view, relative to its <see cref="SuperView"/>. 
 		/// </summary>
-		/// <value>The frame.</value>
+		/// <value>The rectangle describing the location and size of the view, in coordinates relative to the <see cref="SuperView"/>.</value>
 		/// <remarks>
 		/// <para>
-		///    Change the Frame when using the <see cref="Terminal.Gui.LayoutStyle.Absolute"/> layout style to move or resize views. 
+		/// Change the Frame when using the <see cref="Terminal.Gui.LayoutStyle.Absolute"/> layout style to move or resize views. 
 		/// </para>
 		/// <para>
-		///    Altering the Frame of a view will trigger the redrawing of the
-		///    view as well as the redrawing of the affected regions of the <see cref="SuperView"/>.
+		/// If <see cref="LayoutStyle"/> is <see cref="LayoutStyle.Computed"/>, altering the Frame will 
+		/// change the layout style to <see cref="LayoutStyle.Absolute"/> and  <see cref="X"/>, <see cref="Y"/>, <see cref="Width"/> and <see cref="Height"/>
+		/// will be set to the values of the Frame (using <see cref="Pos.PosAbsolute"/> and <see cref="Dim.DimAbsolute"/>).
+		/// </para>
+		/// <para>
+		/// Altering the Frame will eventually (when the view is next drawn) cause the <see cref="LayoutSubview(View, Rect)"/> and <see cref="Redraw(Rect)"/> methods to be called.
 		/// </para>
 		/// </remarks>
 		public virtual Rect Frame {
 			get => frame;
 			set {
 				frame = new Rect (value.X, value.Y, Math.Max (value.Width, 0), Math.Max (value.Height, 0));
+				X = frame.X;
+				Y = frame.Y;
+				Width = frame.Width;
+				Height = frame.Height;
```

Comment:
You are forcing the Pos/Dim to be `Absolute` even they aren't. So, in my opinion the `Frame` setter mustn't set the Pos/Dim values.